server/src/main/java/org/elasticsearch/action/admin/cluster/stats/CCSUsageTelemetry.java

```diff
@@ -60,6 +61,27 @@ public String getName() {
     public static final String ASYNC_FEATURE = "async";
     public static final String WILDCARD_FEATURE = "wildcards";
 
+    // The list of known Elastic clients. May be incomplete.
+    public static final Set<String> KNOWN_CLIENTS = Set.of(
```

Comment:
Have we checked with the #devtools team on what headers they might set?

Reply:
Yes, Kibana would set it to `kibana`, except when using dev console.